inst/private/store_vars_in_python.m

```diff
@@ -1,40 +1,29 @@
-function store_vars_in_python(var_name, varargin)
-  is_list = false;
-  if numel(varargin) >1
-    is_list = true;
-  end
-  
-  for i= 1:numel(varargin)
-    cur_var = var_name
-    x = varargin{i};
-    store_cur_var = true;
-    
-    if(isa(x, 'sym'));
-      pycall('pyevalstoretemp', sprintf(char(x)));
-      
-    elseif(ischar(x))
-      pycall('pyevalstoretemp', sprintf('str("%s")', x));
-      
-    elseif(ismatrix(x) && ~isscalar(x))
-      % Currently only deals with 2-D matrix. TODO: implement for n-D matrix
-      if(size(x,1)==1)
-        x = x';
-      end
-      for i=1:size(x,1)
-        store_vars_in_python(cur_var, x(i));
-      end
-      store_cur_var = false;  % don't call append for this, already done
-      
+function store_vars_in_python (varname, L)
+%varname
+%L
+
+  for i = 1:numel(L)
+    %i
+    x = L{i};
+    if (isa(x, 'sym'));
+      pyobj = pyeval((char(x)));
+      pycall('pystoretemp', pyobj)
+      pyexec([varname '.append(_temp)'])
```

Comment:
> But, we might still need the python compile unless we deal with the
> numpy arrays created even for strings by pycall. I'll probably start
> looking into that now.

eh?  Please give example what you mean.

* strings work fine

* scalar doubles work fine

Nothing else we need to Symbolic right now I think.